metadata/src/main/java/org/apache/kafka/image/loader/SnapshotManifest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.image.loader;
+
+import org.apache.kafka.image.MetadataProvenance;
+
+import java.util.Objects;
+
+
+/**
+ * Contains information about a snapshot that was loaded.
+ */
+public class SnapshotManifest {
+    /**
+     * The source of this snapshot.
+     */
+    private final MetadataProvenance provenance;
+
+    /**
+     * The time in microseconds that it took to load the snapshot.
+     */
+    private final long elapsedNs;
+
+    public SnapshotManifest(
+        MetadataProvenance provenance,
+        long elapsedNs
+    ) {
+        this.provenance = provenance;
+        this.elapsedNs = elapsedNs;
+    }
+
+    public MetadataProvenance provenance() {
+        return provenance;
+    }
+
+    public long elapsedNs() {
```

Comment:
I don't think this is used. Can we remove it?

Reply:
similarly to the log delta one, we need it for DEBUG log4j logging (which was missing, but I will add)